pkg/users/accounts-list.js

```diff
@@ -183,17 +249,77 @@ const mapGroupsToAccount = (accounts, groups) => {
     });
 };
 
-const AccountsList = ({ accountsInfo, current_user, groups }) => {
-    const [currentTextFilter, setCurrentTextFilter] = useState("");
+const GroupsList = ({ groups, accounts }) => {
+    const [isExpanded, setIsExpanded] = useState(false);
+    const columns = [
+        { title: _("Group name"), sortable: true },
+        { title: _("ID"), sortable: true },
+        { title: _("# of users"), sortable: true },
+        { title: _("Accounts") },
+    ];
 
-    const accounts = mapGroupsToAccount(accountsInfo, groups);
+    const sortRows = (rows, direction, idx) => {
+        const sortedRows = rows.sort((a, b) => {
+            const aitem = a.columns[idx];
+            const bitem = b.columns[idx];
+            const aname = a.columns[0];
+            const bname = b.columns[0];
 
-    const filtered_accounts = accounts.filter(account => {
-        if ((account.uid < 1000 && account.uid !== 0) ||
-                 account.shell.match(/^(\/usr)?\/sbin\/nologin/) ||
-                 account.shell === '/bin/false')
-            return false;
+            // current user is always first
+            if (admins.includes(aname.title))
+                return direction === SortByDirection.asc ? -1 : 1;
+            if (admins.includes(bname.title))
+                return direction === SortByDirection.asc ? 1 : -1;
 
+            return (aitem.title).toString().localeCompare(bitem.title.toString());
```

Comment:
`localeCompare` seems to be a bit problematic when sorting by id (and # of users too). What I'm seeing now is order: 1; 10; 100; 1000; 104; ... 11;

Note that this is also used in the accounts table so it's wrong in the code I wrote as well! :)

Reply:
Fixed.